modules/light-clients/10-wasm/client/cli/tx.go

```diff
@@ -0,0 +1,165 @@
+package cli
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/cosmos/cosmos-sdk/version"
+	clienttypes "github.com/cosmos/ibc-go/modules/core/02-client/types"
+	"github.com/cosmos/ibc-go/modules/light-clients/10-wasm/types"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func NewCreateClientCmd() *cobra.Command {
```

Comment:
These commands should not require changes. Any wasm-specific checks should happen in wasm handlers

Reply:
This adds additional commands, such as this:
```
./ibc-go/build/simd tx ibc wasm-client
WASM client transaction subcommands

Usage:
  simd tx ibc wasm-client [command]

Available Commands:
  create       create new wasm client
  misbehaviour submit a client misbehaviour
  update       update existing client with a header

Flags:
  -h, --help   help for wasm-client

Global Flags:
      --chain-id string     The network chain ID
      --home string         directory for config and data (default "/home/admin/.simapp")
      --log_format string   The logging format (json|plain) (default "plain")
      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
      --trace               print out full stack trace on errors

Use "simd tx ibc wasm-client [command] --help" for more information about a command
```

If I understood your comment correctly, the `ibc wasm-client` commands can be merged into `ibc client`. I'll see if that can be done, thanks.

Reply:
removed, thanks